build.gradle

```diff
@@ -18,6 +18,7 @@ allprojects {
 
     repositories {
         jcenter()
+        mavenCentral()
```

Comment:
What is this change for?

Reply:
jcenter was down, this allowed the build to succeed